package.json

```diff
@@ -11,17 +11,17 @@
   "type": "module",
   "exports": {
     ".": {
-      "import": "./dist/cal-heatmap.esm.js",
-      "require": "./dist/cal-heatmap.js",
-      "types": "./src/types.d.ts"
+      "import": "./dist/cal-heatmap.js",
+      "require": "./dist/cal-heatmap.cjs",
+      "types": "./dist/index.d.ts"
```

Comment:
`types` must be the first field in `exports`

Reply:
That's not true.
I checked other libraries... (e.g. https://github.com/toss/slash/blob/main/packages/common/assert/package.json#L56)

Anyway, I will close this pr.
Because, the maintainer is not active for a long time.

Reply:
Here's the docs. Sorry I couldn't find it anywhere outside of Node.JS docs for you, but hopefully that's enough of an authority :)
https://nodejs.org/api/packages.html#community-conditions-definitions

![image](https://github.com/wa0x6e/cal-heatmap/assets/25051063/e6454e50-ab99-4afd-9f7c-07f932046137)

I imagine some variations of environment might be less strict, but you should aim to support the strictest where possible :)

Reply:
I understand what you want to say.
But I think you're misunderstanding about this.
This document mentions the order of "types", "browser", "development", and "production", but does not mention the order of "import", "require", and "types".
This is a different situation.
Check out axios [`package.json`](https://github.com/axios/axios/blob/v1.x/package.json#L6)

Reply:
Their `types` are top 🤷 
<img width="343" alt="image" src="https://github.com/wa0x6e/cal-heatmap/assets/25051063/f8903148-6e2f-4deb-904b-8dbd6acb9cbb">
Anyway, I have no stake in this :) See ya around :)

Reply:
Yes, I told you this to show how my package.json and axios' package.json are different.
Axios' package.json is mentioned in the document, but it has nothing to do with the code I wrote.

Reply:
If I really wrote the code wrong, an error would have occurred :)